pkg/collector/pipeline.go

```diff
@@ -113,6 +130,8 @@ func (p *Pipeline) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	var reports ReportBatch
 	reports.Time = clock.Now()
+	reports.ClientIP = strings.Split(r.RemoteAddr, ":")[0]
```

Comment:
What about IPv6?

Reply:
Done.  Found the handy net.SplitHostPort function that takes care of both IPv4 and IPv6